mne/tests/test_source_estimate.py

```diff
@@ -261,12 +261,12 @@ def test_expand():
         pytest.raises(ValueError, stc.__add__, stc.in_label(labels_lh[0]))
 
 
-def _fake_stc(n_time=10):
+def _fake_stc(n_time=10, Complex=False):
     verts = [np.arange(10), np.arange(90)]
     return SourceEstimate(np.random.rand(100, n_time), verts, 0, 1e-1, 'foo')
 
 
-def _fake_vec_stc(n_time=10):
+def _fake_vec_stc(n_time=10, Complex=False):
```

Comment:
don't use capital letters in variable names. I would suggest `iscomplex=False`

Reply:
I've changed the name to _is_complex_ and updated _latest.inc_